net/ebtables/ebtables.go

```diff
@@ -121,7 +121,7 @@ func getEbtablesVersionString(exec utilexec.Interface) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	versionMatcher := regexp.MustCompile(`v([0-9]+\.[0-9]+\.[0-9]+)`)
+	versionMatcher := regexp.MustCompile(`([0-9]+\.[0-9]+\.[0-9]+)`)
```

Comment:
Maybe clearer with `v?` at the beginning?  And a comment explaining this is actually seen in the wild in both forms.

Agree with the point about a unit test.  Maybe factor this to a `parseVersion(string) (string, error)` function and test that with a few interesting inputs.

Reply:
https://go.dev/play/p/C5_zp5HI8uX